src/SFML.Window/WindowBase.cs

```diff
@@ -421,7 +421,7 @@ protected internal virtual void InternalSetMousePosition(Vector2i position)
         ////////////////////////////////////////////////////////////
         protected internal virtual Vector2i InternalGetTouchPosition(uint Finger)
         {
-            return sfTouch_getPosition(Finger, CPointer);
+            throw new NotImplementedException("Currently not available");
```

Comment:
Are we not implementing it, because there hasn't been a new CSFML release yet with that function included?

Reply:
These internal functions exist because the getPosition overloads for a window, needs the private window CPointer.

Additionally, this is not implemented here, because we haven't created a CSFML 2.6.1 release with the missing function implementation.